### DISCUSSION TITLE
How to automatically approve changes to documentation (and other files)

### DISCUSSION THREAD

**Discussion by joe-polin** [community/community#53327] [Pull Requests]
### Select Topic Area

General

### Body

Like most organizations, we have a strict guidelines around merging pull requests which require someone (other than the author) to approve a change before it can be merged. However, there are some instances where this friction is counter productive:

- Documentation: Since this cannot lead to any code breakages, we'd prefer to allow people to update documentation with as little friction as possible. Right now, it's not worth the hassle to fix typos or add a new "gotcha." 
- Experimental directories: To combat the pattern of engineers spending weeks developing (without rebasing) in their own branch, it would be nice to have an experimental folder in our codebase that similarly requires very little (or no) approval to merge.

I've been looking into enabling this for our repository, but it isn't as straightforward as I hoped. So far, it seems the best plan is to:

- Add my desired patterns to the CODEOWNERS file, and have something like:
```
**/README*      @autoapprover
experimental    @autoapprover
```
- Create an autoapprover bot that automatically approves anything that comes its way.

Does that seem like the most straightforward approach?

**Comment by queenofcorgis** [comment]
Hey Joe 👋🏼 , 

Thanks for asking this question. I reviewed this as well as a member of our Pull Requests team and she said she would suggest the exact same approach. 

Though I welcome anyone else with other suggestions!
